templates/vue/src/utils/errorHelper.js

```diff
@@ -0,0 +1,27 @@
+    export default class ErrorHelper {
+    /**
+   * Return true if response is an error
+   *
+   * @param {Object} response
+   *
+   * @return {Boolean}
+   */
+    static isError(response) {
+        return response.status !== 200
```

Comment:
@jaskwong Should we support other successful response codes in the 200 range?

Reply:
good call, I'll update it, but right now this function is never called so I might delete it later

Reply:
I've removed this function since I agree with @rlnsy and also it wasn't being used.